app/controllers/admin/display_ads_controller.rb

```diff
@@ -6,13 +6,13 @@ class DisplayAdsController < Admin::ApplicationController
 
     def index
       @display_ads = DisplayAd.order(id: :desc)
-        .joins(:organization)
-        .includes([:organization])
         .page(params[:page]).per(50)
 
       return if params[:search].blank?
 
-      @display_ads = @display_ads.where("organizations.name ILIKE :search", search: "%#{params[:search]}%")
+      @display_ads = @display_ads.
```

Comment:
For anyone else who wonders, this is a small table (currently 29 items on DEV), so even inefficient queries are probably ok.